packages/ui/src/lib/settingsNavItem/SettingsNavItem.svelte

```diff
@@ -66,7 +69,12 @@ function click(): void {
     class:hover:border-[var(--pd-secondary-nav-text-hover-bg)]={!selected}>
     <span class="group-hover:block flex flex-row items-center" class:capitalize={!child}>
       {#if icon}
-        <Fa class="mr-4" icon={icon} />
+        {#if isFontAwesomeIcon(icon)}
+          <Fa class="mr-4" {icon} />
+        {:else}
+          <!-- svelte-ignore svelte_component_deprecated -->
+          <svelte:component this={icon} size={'14'} class="mr-4"/>
```

Comment:
<icon size=.../> might work ?

Reply:
(I think you might need to edit your comment as the `<` and `>` are stripped from the rendering

Reply:
hihi yes